avocado/utils/linux_modules.py

```diff
@@ -189,11 +190,14 @@ def module_is_loaded(module_name):
 
     :param module_name: Name of module to search for
     :type module_name: str
-    :return: True is module is loaded
+    :return: True if module is loaded
     :rtype: bool
     """
     module_name = module_name.replace('-', '_')
-    return bool(loaded_module_info(module_name))
+    for line in genio.read_file("/proc/modules").splitlines():
+        if line.split()[0] == module_name:
```

Comment:
minor optimization, you are only interested in the first part, so how about using `split(None, 1)` to avoid splitting the line after first split.